### PR TITLE
Add old proto rules

### PR DESCRIPTION
**Pull request by Tatskaari**

### CODE REVIEW

File: build_defs/legacy.build_defs [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+"""
+
+if "go" in CONFIG.PROTO.LANGUAGES:
+    subinclude("///go//build_defs:go")
+if "py" in CONFIG.PROTO.LANGUAGES:
+    subinclude("///python//build_defs:python")
+if "java" in CONFIG.PROTO.LANGUAGES:
+    subinclude("///java//build_defs:java")
+if "cc" in CONFIG.PROTO.LANGUAGES:
+    subinclude("///cc//build-defs:cc")
```
Comment (goddenrich) [inline build_defs/legacy.build_defs]: I think these should be CONFIG.LANGUAGES. Similarly for other CONFIG.PROTO values

**Review by goddenrich — COMMENTED** [review]
Should we be setting a version? would v1.0.0 make sense like we have v1 for the old rules in the python plugin?

**Comment by goddenrich** [comment]
We should also probably change the README if we are going down this approach

File: build_defs/legacy.build_defs (hunk 1 above)
Comment (Tatskaari) [reply build_defs/legacy.build_defs]: That wasn't what I had in mind but maybe you're right. Do you think we should be using the old `[Proto]` section? I think that might actually work but I was only keeping those around to aide `plz init plugin`. Maybe we can discuss in standup.

**Review by goddenrich — APPROVED** [review]

**Pull request merged**